source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -12,7 +13,28 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - >= 4.0
```

Comment:
We haven't released 4.0 yet, and its release date is currently unknown, so let's keep it off of this list for now.